src/fromager/sdist.py

```diff
@@ -195,7 +195,16 @@ def handle_requirement(
                 | build_sdist_dependencies,
             )
             try:
-                sources.build_sdist(ctx, req, sdist_root_dir)
+                find_sdist_result = finders.find_sdist(
+                    ctx.sdists_builds, req, resolved_version
+                )
+                if not find_sdist_result:
+                    logger.info(f"{req.name}: already resolved sdist version not found")
```

Comment:
We don't need this much verbosity. Let's focus on logging things that are different from the expected default behavior. In this case, for example, it would be when we do find an sdist and so we are not going to rebuild it.

Reply:
Does `sdist not found, building sdist` sound good?

Reply:
I think you want to remove this log message entirely.

Reply:
Okay